duf: add page (#8171)
tldr-pages/tldr: pages/common/duf.md

# duf

> > Disk Usage/Free Utility - a better 'df' alternative.

Reviewer: ```suggestion
> Disk Usage/Free Utility.
```

> More information: <https://github.com/muesli/duf>.

- List accessible devices:

`duf`

> - List everything (including pseudo, duplicate, inaccessible file systems):

Reviewer: ```suggestion
- List everything (such as pseudo, duplicate or inaccessible file systems):
```

`duf --all`

> - Only list specific devices & mount points:

Reviewer 2: ```suggestion
- Only list specific devices or mount points:
```

> - Only list specific devices or mount points:

Reviewer: ```suggestion
- Only show specified devices or mount points:
```

> `duf /home /some/file`

Reviewer 2: ```suggestion
`duf {{/path/to/directory1 /path/to/directory2}}`
```

> - Sort the output:

Reviewer 2: ```suggestion
- Sort the output by size:
```

> - Sort the output:

Reviewer: ```suggestion
- Sort the output by a specified criteria:
```

> `duf {{/path/to/directory1 /path/to/directory2}}`

Reviewer 3: ```suggestion
`duf /{{path/to/directory1 path/to/directory2 ...}}`
```

If leading slash is mandatory you can put it before `{{`.

> `duf /{{path/to/directory1 path/to/directory2 ...}}`

Reviewer: ```suggestion
`duf {{path/to/directory1 path/to/directory2 ...}}`
```

> `duf --sort size`

Reviewer 2: ```suggestion
`duf --sort {{size}}`
```

Are there any other options we can use here except size?

Author: > Valid keys are: mountpoint, size, used, avail, usage, inodes, inodes_used, inodes_avail, inodes_usage, type, filesystem.

Author: Thanks. I've added it in the last commit.
